### PR TITLE
Logs cc session exception to Sentry

### PR DESCRIPTION
**Pull request by shubham1g5**
## Technical Summary

Trying to get more info on https://dimagi-dev.atlassian.net/browse/USH-2576

### Automated test coverage

N/A

### QA Plan

N/A

### Migrations

- [x] The migrations in this code can be safely applied first independently of the code.

### Special deploy instructions

- [x] This PR can be deployed after merge with no further considerations.

### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations.

### Review

- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change.

### CODE REVIEW

File: src/main/java/org/commcare/formplayer/services/MenuSessionRunnerService.java [hunk 1]
```diff
@@ -249,84 +249,91 @@ public BaseResponseBean advanceSessionWithSelections(MenuSession menuSession,
...
+                    detailSelection,
                     offset,
                     searchText,
                     sortIndex,
                     queryData,
                     casesPerPage,
                     smartLinkTemplate
             );
+            restoreFactory.cacheSessionSelections(menuSession.getSelections());
+        } catch (CommCareSessionException ccse) {
```
Comment (shubham1g5) [inline src/main/java/org/commcare/formplayer/services/MenuSessionRunnerService.java:326]: Diff on this is not great, this catch is the main change here.

Comment (snopoke) [reply src/main/java/org/commcare/formplayer/services/MenuSessionRunnerService.java:326]: how does this end up in Sentry?

Comment (shubham1g5) [reply src/main/java/org/commcare/formplayer/services/MenuSessionRunnerService.java:326]: all the notifications tagged as errors gets logged through [this method](https://github.com/dimagi/formplayer/blob/master/src/main/java/org/commcare/formplayer/application/MenuController.java#L149).

**Comment by shubham1g5** [comment]
@snopoke bumping this for review.

**Review by snopoke — APPROVED** [review]

**Pull request merged**